packages/cli-kit/src/public/node/api/partners.ts

```diff
@@ -2,7 +2,14 @@ import {graphqlRequest, GraphQLVariables, GraphQLResponse} from './graphql.js'
 import {partnersFqdn} from '../context/fqdn.js'
 import {setNextDeprecationDate} from '../../../private/node/context/deprecations-store.js'
 import {gql} from 'graphql-request'
+import {createRequire} from 'module'
 
+const require = createRequire(import.meta.url)
+// eslint-disable-next-line import/no-extraneous-dependencies
+const limiter = require('limiter')
+
+// Allow 1 request to partners every 110ms (the limit is 10 requests per second, adding a 10% margin)
+const partnersLimiter = new limiter.RateLimiter({tokensPerInterval: 1, interval: 110})
```

Comment:
Can't we do something like `limiter.RateLimiter({tokensPerInterval: 10, interval: 'second'})`? This would allow multiple requests below the limit of 10 to not be affected by the limit.